proposals/using-alias-types.md

```diff
@@ -38,20 +38,22 @@ The sections that do need updating are:
 + The namespace_name referenced by a using_namespace_directive is resolved in the same way as the namespace_or_type_name referenced by a using_alias_directive. Thus, using_namespace_directives in the same compilation unit or namespace body do not affect each other and can be written in any order.
 ```
 
+```diff
++ It is illegal for a using alias type to be a nullable reference type.  e.g. `using X = string?;` is not legal.
```

Comment:
Does this need to specify top-level in some fashion? Ie, `List<string?>` will still be legal to alias.

Reply:
Happy to add that.